scapy/main.py

```diff
@@ -391,7 +399,11 @@ def init_session(session_name,  # type: Optional[Union[str, None]]
     global SESSION
     global GLOBKEYS
 
-    scapy_builtins = {k: v for k, v in six.iteritems(importlib.import_module(".all", "scapy").__dict__) if _validate_local(k)}  # noqa: E501
+    scapy_builtins = {k: v
+                      for k, v in six.iteritems(
+                             importlib.import_module(".all", "scapy").__dict__
```

Comment:
flake8 does not like this line (see https://travis-ci.com/secdev/scapy/jobs/270916974#L656)